test/test_generators.py

```diff
@@ -1,34 +0,0 @@
-# -*- coding: utf-8 -*-
-"""
-Unit tests for mathics.builtins.numbers.algebra
-"""
-from test.helper import check_evaluation, evaluate, evaluate_value
-
-
-def setup_module(module):
-    """Load pymathics.graph"""
-    assert evaluate_value('LoadModule["pymathics.graph"]') == "pymathics.graph"
-    evaluate("SortList[list_] := Sort[Map[Sort, list]]")
-
-
-def test_connected_components():
```

Comment:
Where did this test go?

Reply:
In master I am seeing this test twice. Once in test_algorithms.py, and once in test_generators. 

But code in  "algorithms" has been moved to components and other places. And there was a test in that which should have been in the non-existent `test.test_base`. 

Commit fbd8837 addresses both of these.